main.js

```diff
@@ -56,3 +56,22 @@ form.addEventListener('submit', (event) => {
     }, 1000);
   }
 });
+
+const time = document.querySelector('form');
+const fullNameInput = document.querySelector('#full-name');
+const eInput = document.querySelector('#email');
+const messageInput = document.querySelector('#message');
+
+time.addEventListener('submit', (event) => {
+  event.preventDefault();
+
+  const fullName = fullNameInput.value;
+  const email = eInput.value;
+  const message = messageInput.value;
+
+  localStorage.setItem('fullName', fullName);
+  localStorage.setItem('email', email);
+  localStorage.setItem('message', message);
+
+  form.reset();
+});
```

Comment:
- [ ] Kindly make sure when the form is submitted to the email the way it was before and when I refresh the page the data served previously needs to be populated in the form. Below you will find a screencast that gives you more clarification

https://user-images.githubusercontent.com/65068771/233765325-9e30965a-308b-4d5c-924a-a3c32532e42e.mov